Intelipro/reja
# Feat: Implement Error Boundary pages

#### What does this PR do?
It implements error boundary page for unknown links and caught app errors

#### Description of Task to be completed?
- [x] implement page for the caught  errors
- [x] Implement page for unknown app links

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ft-error-boundary-page`
- Install all dependencies `npm install`
- Edit the URL on any page to a random value to see the error page
- The error page only appear when errors are caught on the app

#### Any background context you want to provide?
Error boundaries are React components that catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI instead of the component tree that crashed

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/43441107/77312969-07736d00-6d14-11ea-9db3-089e062acdef.png)
#### Questions:
